Dockerfile

```diff
@@ -11,8 +11,8 @@ RUN pip install arcgis --no-deps
 WORKDIR /kartograafr/
 COPY . /kartograafr/
 
-# Set up log file directories
-RUN mkdir -p -v /var/log/kartograafr/courses
```

Comment:
If I understand correctly, the `kartograafr` directory is now a mount point for a volume, so this has been changed to make the necessary subdirectory within it.  I suspect that's if it's necessary.  Since it's using a volume for these logs now, the logs will be preserved from one pod run to another, so the directory may already be there from a previous run.

The logs hadn't been written to a volume before because one of the design goals would be to store the logs for each course within the "files" section of each course in Canvas.  Having a persistent volume for the logs doesn't preclude that, of course, it just wasn't deemed very important.

I don't remember: Do we already have code in the project to dispose of old log files?

Reply:
I don't remember seeing anything that removes the old files, but yes, that's something that would make sense to include.

Reply:
Right now it seems like the email will send all of the logs available for a course, including ones from previous runs. Is that the expected behavior? If we only want results from the current run, we can probably get rid of the volume and instead write the log files to the `tmp` directory and have them disappear with the pod. We decided to go the volume route because the cost was insignificant and it was nice to keep the setup consistent with local development.

Reply:
We don't want to inundate anyone with a large amount of log files.  So, the code should be changed to send only the most recent log or we should stick with ephemeral storage.

I agree it would be good to keep the logs around, for support purposes.  So maybe updating the code to send only the most recent one and to also delete old ones is the best thing to do.

Reply:
Okay, I've changed this as I've discussed with you and @zqian to create the directories for log files under `/tmp`. Please let me know what you think.

Also, I had said earlier that the emails were sending results from multiple log files, but upon further inspection, I think that's incorrect and only the results of the latest run are sent.